.devcontainer/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM metaseqopt.azurecr.io/metaseq-opt-singularity:latest
```

Comment:
You would likely have to change the base image.
I think this image _is_ public, but there aren't guarantees about preserving it. 
Also has some other code for private cluster that you would not need and adds extra complexity